packages/vscode-extension/src/debug/npmInstallHandler.ts

```diff
@@ -16,3 +32,47 @@ export async function runNpmInstallAll(projectRoot: string): Promise<void> {
     terminal.sendText("npm run installAll");
   }
 }
+
+export async function automaticNpmInstallHandler(): Promise<void> {
+  const state = globalStateGet("automaticNpmInstall", false);
+  if (state) {
+    globalStateUpdate("automaticNpmInstall", false);
+    const configuration = getConfiguration(ConfigurationKey.AutomaticNpmInstall);
+    if (configuration && ext.workspaceUri !== undefined) {
+      VS_CODE_UI.showMessage(
+        "info",
+        StringResources.vsc.handlers.automaticNpmInstall,
+        false,
+        StringResources.vsc.handlers.disableAutomaticNpmInstall
+      ).then((selection) => {
+        if (
+          selection.isOk() &&
+          selection.value === StringResources.vsc.handlers.disableAutomaticNpmInstall
+        ) {
+          vscode.commands.executeCommand(
+            "workbench.action.openSettings",
```

Comment:
Which scope of settings is opened by default, user or workspace?

Reply:
User. Same behavior as change `Default Project Root Directory` setting.